core/server/webhooks-events.js

```diff
@@ -0,0 +1,79 @@
+var _ = require('lodash'),
+    errors = require('./errors'),
+    events = require('./events'),
+    logging = require('./logging'),
+    models = require('./models'),
+    https = require('https'),
```

Comment:
We have a request utility in `utils/request.js` that should help to make this code much easier to write. We currently only use it in one place, and use http/https everywhere else. This can stay as is no requirement to rewrite, just mentioning that the functionality is there.

Also, again purely so you know, utils/request.js should be lib/request.js 😁  as per https://github.com/TryGhost/Ghost/issues/9178